### ISSUE TITLE
Feature: Add ability to change background and font colors on web app.

### ISSUE THREAD

**Issue by muxall** [muxall/RPTCQ-LCD#149]
The yellow background color and blue font color have been reported hard to read for a person with cataracts.

**Comment by PhilSmith46** [comment]
Agree, I have cataracts, but not too bad yet and it's a little difficult to read at times,

**Comment by muxall** [comment]
Yellows, greens and blue seems to be the problem for me if on the same page or graph.

**Comment by muxall** [comment]
blues, purples, whites,blacks, grays. My color blind issues are reds, greens, browns
